diff --git a/os/src/sbi.rs b/os/src/sbi.rs
index dc5bbe3..c1d2403 100644
--- a/os/src/sbi.rs
+++ b/os/src/sbi.rs
@@ -4,10 +4,10 @@
 
 use core::arch::asm;
 
-const SBI_SET_TIMER: usize = 0;
+const SBI_SET_TIMER: usize = 0x54494D45;
 const SBI_CONSOLE_PUTCHAR: usize = 1;
 const SBI_CONSOLE_GETCHAR: usize = 2;
-const SBI_SHUTDOWN: usize = 8;
+const SBI_SHUTDOWN: usize = 0x53525354;
 
 /// general sbi call
 #[inline(always)]
